### PR TITLE
[Profiles] search and update profile actions assigned to hotkeys

### PR DESCRIPTION
**Pull request by Commodore64user**
### what's new

* minor addition to Profile plugin to update profile actions assigned to hotkeys

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/12642)

### CODE REVIEW

**Comment by hius07** [comment]
It should be the part of https://github.com/koreader/koreader/pull/12484.

File: plugins/profiles.koplugin/main.lua [hunk 1]
```diff
@@ -435,6 +435,8 @@ function Profiles:updateProfiles(action_old_name, action_new_name)
     end
     if self.ui.gestures then -- search and update the profile action in assigned gestures
         self.ui.gestures:updateProfiles(action_old_name, action_new_name)
+    elseif self.ui.hotkeyshortcuts then -- search and update the profile action in assigned keyboard shortcuts
```
Comment (Commodore64user) [inline plugins/profiles.koplugin/main.lua:438]:
```suggestion
    elseif self.ui.hotkeys then -- search and update the profile action in assigned keyboard shortcuts
```

File: plugins/profiles.koplugin/main.lua [hunk 2]
```diff
@@ -435,6 +435,8 @@ function Profiles:updateProfiles(action_old_name, action_new_name)
...
+        self.ui.hotkeyshortcuts:updateProfiles(action_old_name, action_new_name)
```
Comment (Commodore64user) [inline plugins/profiles.koplugin/main.lua:439]:
```suggestion
        self.ui.hotkeys:updateProfiles(action_old_name, action_new_name)
```

**Comment by Commodore64user** [comment]
image

**Pull request merged**